### ISSUE TITLE
Set param.requires_grad to False 

### ISSUE THREAD

**Issue by wanghonghui1998** [megvii-model/DetNAS#34]
Hello, great work! 
I have a question in this code. Why do you set param.requires_grad to False? Won't it cause a runtime error when calling loss.backward()? Thanks for your answer. 
https://github.com/megvii-model/DetNAS/blob/aa92a90604c870fcb7e3ea9f60d16e6f107454d9/maskrcnn_benchmark/engine/trainer.py#L112-L113
